### PR TITLE
Code improvements for Pull Request

### LINKED ISSUE #603: Add PHP Language Article

### LINKED ISSUE #604: Add Even Odd in PHP Article

### PR DESCRIPTION
**Pull request by Faiznurullah**
Thanks for helping us build up our knowledge base! 
What kind of pull request have you brought us today?
Please make sure to familiarize yourself with the [Contributing Guide][2].

## Pull Request Type

Please fill out this form in the following way:

```
[x] Right :)
[ x ] Wrong :(
```

Please specify the type of pull request (please only check one of these):

- [x] Sample Program Documentation
- [x] Language Documentation
- [ ] Other (explain in the Notes section)

This pull request *must* have a corresponding issue number. If you have
not already done so, please [create a new issue][1].

- I fixed #603 
- I fixed #604 

## Notes

Feel free to share any details you feel are relevant.

[1]: https://github.com/TheRenegadeCoder/sample-programs-website/issues/new/choose
[2]: https://github.com/TheRenegadeCoder/sample-programs-website/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by Faiznurullah** [comment]
Please Give Me Label Hacktoberfest-Accepted Thanksyou

File: docs/projects/sleep-sort/python/index.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ tags:
 title: Sleep Sort in Python
 ---
 
+
```
Comment (rzuckerm) [inline docs/projects/sleep-sort/python/index.md:15]: Please revert this change. This PR should not contain anything from the `docs` directory

File: sources/programs/even-odd/php/how-to-implement-the-solution.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+// Test the function with a number
+$number = 7; // You can change this number to test different values
+
+// Call the function to check if the number is even or odd
+checkEvenOdd($number);
+?>
+
+```
```
Comment (rzuckerm) [inline sources/programs/even-odd/php/how-to-implement-the-solution.md:24]: The code that should be written about is [here](https://github.com/TheRenegadeCoder/sample-programs/blob/main/archive/p/php/even-odd.php). Please write about that.

**Review by rzuckerm — CHANGES_REQUESTED** [review]

File: docs/projects/sleep-sort/python/index.md (hunk 1 above)
Comment (Faiznurullah) [reply docs/projects/sleep-sort/python/index.md:15]: Its from pull origin fuck, i dont change anything in folder it

**Comment by Faiznurullah** [comment]
I have fixed everything, please merge and give me the hacktoberfest-accepted label

**Comment by Faiznurullah** [comment]
Fuck You
